cookbooks/fb_ipset/files/default/ipset-init

```diff
@@ -18,24 +18,23 @@
 ### END INIT INFO
 
 # Source function library.
+# shellcheck disable=SC1091
 . /etc/init.d/functions
 
 IPSET=ipset
 IPSET_BIN=/usr/sbin/$IPSET
 IPSET_DATA=/etc/sysconfig/$IPSET
 
 if [ ! -x ${IPSET_BIN} ]; then
-    echo -n $"${IPSET}: ${IPSET_BIN} does not exist."; warning; echo
+    printf '%s: %s does not exist.' ${IPSET} ${IPSET_BIN}; warning; echo
```

Comment:
Ugh I hate multiline stuff like this. Might I recommend, while your'e fixing this.
```suggestion
    printf '%s: %s does not exist.' ${IPSET} ${IPSET_BIN}
    warning
    echo
```